cfpb/consumerfinance.gov
# Bump cfpb-chart-builder to fix vulnerabilities

## Changes

- Bump cfpb-chart-builder to ``6.4.1`
- Bump `highcharts` to `7.2.2` to fix dependabot vulnerabilities.


## How to test this PR

1. Pull branch and build.
2. Visit charts on http://localhost:8000/owning-a-home/explore-rates/, http://localhost:8000/data-research/mortgage-performance-trends/mortgages-30-89-days-delinquent/, and http://localhost:8000/data-research/consumer-credit-trends/auto-loans/ and compare to production. There should be no change in behavior.
